clients/gtest/solver/getrf_gtest.cpp

```diff
@@ -73,17 +73,22 @@ namespace
             switch(GETRF_TYPE)
             {
             case GETRF:
-                return !strcmp(arg.function, "getrf");
+                return !strcmp(arg.function, "getrf") || !strcmp(arg.function, "getrf_bad_arg");
             case GETRF_BATCHED:
-                return !strcmp(arg.function, "getrf_batched");
+                return !strcmp(arg.function, "getrf_batched")
+                       || !strcmp(arg.function, "getrf_batched_bad_arg");
             case GETRF_STRIDED_BATCHED:
-                return !strcmp(arg.function, "getrf_strided_batched");
+                return !strcmp(arg.function, "getrf_strided_batched")
+                       || !strcmp(arg.function, "getrf_strided_batched_bad_arg");
             case GETRF_NPVT:
-                return !strcmp(arg.function, "getrf_npvt");
+                return !strcmp(arg.function, "getrf_npvt")
+                       || !strcmp(arg.function, "getrf_npvt_bad_arg");
             case GETRF_NPVT_BATCHED:
-                return !strcmp(arg.function, "getrf_npvt_batched");
+                return !strcmp(arg.function, "getrf_npvt_batched")
+                       || !strcmp(arg.function, "getrf_npvt_batched_bad_arg");
             case GETRF_NPVT_STRIDED_BATCHED:
-                return !strcmp(arg.function, "getrf_npvt_strided_batched");
+                return !strcmp(arg.function, "getrf_npvt_strided_batched")
+                       || !strcmp(arg.function, "getrf_npvt_strided_batched_bad_arg");
```

Comment:
Curious what the assembly is on all these, it isn't constexpr so can the switch be optimized out?